subsys/bluetooth/controller/Kconfig

```diff
@@ -215,15 +215,6 @@ config BT_CTLR_ISO_TX_BUFFERS
 	  Set the number of Isochronous Tx PDUs to be queued for transmission
 	  in the controller.
 
-config BT_CTLR_ISO_TX_BUFFER_SIZE
-	int "Isochronous Tx buffer size"
-	depends on BT_CTLR_ADV_ISO || BT_CTLR_CONN_ISO
-	range 1 4095
-	default 27
-	help
-	  Size of the Isochronous Tx buffers and the value returned in HCI LE
-	  Read Buffer Size V2 command response.
-
```

Comment:
Revert this back, as this is the value used to return ISO_Data_Packet_Length  in HCI LE Read Buffer Size v2 command response